nvc/hash.go

```diff
@@ -0,0 +1,20 @@
+package nvc
+
+import (
+	"fmt"
+	"hash/fnv"
+)
+
+func Hash2String(hash Hash) string {
+	return fmt.Sprintf("%016x", uint64(hash))
```

Comment:
Move this to the `String` method and delete the `Hash2String` function

Reply:
:boom: